discord/interactions.py

```diff
@@ -306,12 +318,12 @@ def is_component(self) -> bool:
         return self.type == InteractionType.component
 
     @utils.cached_slot_property("_cs_channel")
+    @utils.deprecated("Interaction.channel", "2.7", stacklevel=4)
     def cached_channel(self) -> InteractionChannel | None:
-        """The channel the
-        interaction was sent from.
+        """The cached channel the interaction was sent from.
```

Comment:
```suggestion
        """The cached channel from which the interaction was sent.
```
Never end with a preposition